example/fitzhugh_nagumo.jl

```diff
@@ -167,17 +128,18 @@ if false
     plot(Yobs[1:2],"o:")
 
     i = 4
-    v = Vec(Matrix(L)\Vector(Yobs.yy[i+1]))
+    v = SVector(Matrix(L)\Vector(Yobs.yy[i+1]))
     P° = PBridgeProp(Ptrue, Ytrue[i]..., Yobs.tt[i+1], v, Ytrue[i+2]..., L,si^2*I, Ptrue.a)
-    B = eulerb(sample(tt[m*(i-1)+1:m*(i+1)+1] , Wiener{Vec{2,Float64}}()),P°)
+    B = bridge(sample(tt[m*(i-1)+1:m*(i+1)+1] , Wiener{SVector{2,Float64}}()),P°)
```

Comment:
Here I am not sure yet what is the right way to factor the code of euler methods and euler methods where the last point is singular and therefore taken from a fixed value